test/engines/puppeteer.js

```diff
@@ -135,6 +135,66 @@ describe("Puppeteer", function() {
         }
     });
 
+    it("should render a TIFF", async () => {

```

Comment:
if we are adding these more formats please generalize this code into a function with two params type and expected magic string, otherwise we have too much boiler plate code